### ISSUE TITLE
full version dataset: video files lack

### ISSUE THREAD

**Issue by xz316** [SparkleXFantasy/MM-Det#18]
Hello, may I ask if the training set mentioned in your paper is composed of 1,973 AI videos generated by stable video diffusion and 1,000 real videos? I have downloaded your **Full Version**. Among them, there are 1,000 videos after decompression of the data in the "youtube" folder. The 0_real file in the "zeroscope" folder should refer to the test video Internvid-10M, and there are 800 videos after decompression. There are a total of 2,750 real videos you mentioned. Judging from the download results, only 1,800 are available. How can the rest be obtained? For AI videos, the downloaded stable video diffusion only has 1,800 videos. How can the 1,973 videos you mentioned be obtained?

When decompressing the above two **0_real.zip** files, there are data errors as shown in the picture. Will such errors have any impact?

I would be most grateful if you could help answer these questions. Thank you very much!

![Image](https://github.com/user-attachments/assets/f1ad9859-78ba-4ddd-b9b1-5721905c47c4)

![Image](https://github.com/user-attachments/assets/94a29ccc-3ad1-47bb-84b5-d0fcd098ee1f)

**Comment by xz316** [comment]
Hello, I just tried it. These erroneous videos can't be opened normally and the files have been damaged. Could you re-upload these split-volume compressed files?

![Image](https://github.com/user-attachments/assets/523ac698-82ee-4960-84d7-19a2a4f32887)

**Comment by SparkleXFantasy** [comment]
Hi. Thank you for your attention. We provide the following solutions you may concern.
> Full Version of Videos

For real videos, the testing sets consists of videos in `zeroscope`, `opensora` and `videocrafter1`. You can obtain videos in these folders.
For fake videos, the `stablevideodiffusion` dataset contains the exact 1,800 videos. (We clean the dataset and update the number in the final version of the paper.)

> Corrupted Videos

We notice that some videos you demonstrated fail to display and there are errors in the unzip process. We use the following unzip procedures in the **Linux** environment.
```
cd $ZIP_ROOT$
zip -s 0 0_real.zip --out 0_real_total.zip
unzip 0_real_total.zip
```
We find videos under this procedure can be displayed in both Windows and Linux environment, and recommend you to try this. If you meet with further problems, please feel free to provide more information.

**Comment by xz316** [comment]
@SparkleXFantasy 

Thank you for your reply. I'll try decompressing it from the **Linux** system again. Then could you send me your final draft of the paper? I would like to see the description in the final version. My email address is cx125829@163.com.

Overall, the training set consists of 1,000 videos from youtube and 1,800 videos from stable video diffusion. The test set consists of 1,750 videos from internvid and videos from 7 different generators. Is my understanding like this correct?

**Comment by SparkleXFantasy** [comment]
@xz316 Hi.

> The final drafter

The paper is available at [https://arxiv.org/pdf/2410.23623v2](https://arxiv.org/pdf/2410.23623v2).

> The composition of the dataset

You understanding is right. The training dataset contains 1,000 real videos from `youtube` and 1,800 fake videos from `stable video diffusion`. The testing dataset contains 1,750 real videos from `Internvid-10M` and fake videos from 7 generators.

**Comment by xz316** [comment]
@SparkleXFantasy Ok. Thank you very much.
